test/tests/ver-1ha/tests

```diff
@@ -0,0 +1,27 @@
+[Tests]
+    design = 'ADMatReaction.md'
+    issues = '#148'
+    [ver-1ha]
+      type = Exodiff
+      input = 'ver-1ha.i'
+      exodiff = ver-1ha_out.e
+      requirement = 'The system shall be able to model a convective outflow problem and calculate the pressure and concentration of the gas in the second and third enclosure.'
+      verification = 'ver-1ha.md'
+    []
```

Comment:
I don't think we need an exodiff test for this case, the csv file test it well already. 
If you have a reason for using the exodus file, I'd want to hear it, but I think it's redundant here.
 ```suggestion
```

Reply:
Agreed, removed.